client/my-sites/guided-transfer/host-credentials-page/pressable.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import {
+	Email,
+	Password,
+	CreateAccountTip,
+	SubmitSection,
+	WPOrgURL,
+} from './fields';
+
+class Pressable extends Component {
```

Comment:
Agreed this makes sense to be a stateless functional component. Don't think it's a blocker here though, let's do it in a separate PR along with the Bluehost and SiteGround forms